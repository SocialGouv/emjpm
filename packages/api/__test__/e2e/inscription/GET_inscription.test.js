//

const request = require("supertest");

const { knex } = global;
jest.setMock("@emjpm/api/db/knex", knex);

const server = require("@emjpm/api/app");

beforeAll(async () => {
  await knex.migrate.latest();
  await knex.seed.run();
});


test("should return list of tis by region", async () => {
  const response = await request(server).get("/api/v1/inscription/tis");
  expect(response.body).toMatchSnapshot();
  expect(response.status).toBe(200);
});
