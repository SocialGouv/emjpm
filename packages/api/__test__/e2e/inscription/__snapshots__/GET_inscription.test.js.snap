// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should return list of tis by region 1`] = `
Array [
  Object {
    "id": 1,
    "nom": "ti arras",
    "region": "Hauts-de-France",
  },
  Object {
    "id": 2,
    "nom": "ti paris",
    "region": "ÃŽle-de-France",
  },
]
`;
